src/components/Reports/ReportsPageRanking/ReportsPageRanking.js

```diff
@@ -95,7 +95,7 @@ class ReportsPageRanking extends React.Component {
                     {item.name}
                   </a>
                   <p className="text-ranking">
-                    <strong>{item.totalVisits}</strong>{' '}
+                    <strong>{item.totalVisitors}</strong>{' '}
                     <FormattedMessage id="reports_pageranking.total_visits" />
```

Comment:
What do you think about:

```suggestion
                    <FormattedMessage id="reports_pageranking.visitors_count" />
```

Reply:
yes! when i think change when have the content, but can change now